modules/core/src/test/java/org/apache/ignite/internal/client/thin/ThinClientNonTransactionalOperationsInTxTest.java

```diff
@@ -95,4 +97,41 @@ private void checkThinClientCacheOperation(IgniteClient client, Consumer<ClientC
         assertTrue(cache.containsKey(1));
         assertFalse(cache.containsKey(2));
     }
+
+    /** */
+    @Test
+    public void testThinClientCacheRemove() throws Exception {
+        startGrid(0);
+
+        IgniteClient client = Ignition.startClient(new ClientConfiguration().setAddresses(Config.SERVER));
```

Comment:
use try-with-resources construction for creating IgniteClient. Let's also add it for `testThinClientCacheClear`.